src/Component/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { GoogleLogout } from 'react-google-login';
+import { useNavigate } from 'react-router-dom';
+
+function Display() {
+  const navigate = useNavigate();
+  const logout = (res) => {
+    navigate('/');
+  };
```

Comment:
Will this clean the session as well?

Reply:
How are we maintaining the session for user?

Reply:
Yes This also clean session. And right now i am working on Low level design Of how to maintain session